README.md

```diff
@@ -194,483 +211,140 @@ CommandsBefore:
-  DemoTopic1:
-    Type: 'AWS::SNS::Topic'
-    Properties:
-      DisplayName: Hello 123
-  DemoTopic2:
-    Type: 'AWS::SNS::Topic'
-    Properties:
-      DisplayName: Hello 456
-  SomeOtherResource:
-    Type: 'AWS::SNS::Topic'
-    Properties:
-      DisplayName: non-custom
-Metadata: # ... elided ...
 ```
-
-TODO: more examples and cover the details
-TODO: This section is best explained with an worked example / executable demonstration
-
-### Working with *Non*-CloudFormation YAML
-
-`iidy render` can be used to invoke the pre-processor on any YAML
-document. All features are available other than [`custom resource
-templates`](#custom-resource-templates). See `iidy render --help` for
-some more information.
-
-### Converting Deployed Stacks to `iidy`
-Converting current stacks to `iidy` can be done through the `iidy convert-stack-to-iidy` command,
-which accepts 2 parameters: `stackname` (the stack name of the project that you'd want to convert) and
-an `outputDir` (path where the output will be stored). For example:
-
-```Shell
-iidy convert-stack-to-iidy my-cloudformation-stack-1 ./infrastructure
-
-# this should generate 3 files:
-# * _original-template.json
-# * stack-args.yaml
-# * cfn-template.yaml
+? MFA token for arn:aws:iam::002682819933:mfa/example.user: ____
 ```
 
-### `iidy` Environment Variables
+If you've assumed a profile prior to running iidy and want to it to ignore
+what's specified as `Profile` in `stack-args.yaml` and instead use `AWS_*`
+environment variables, set the CLI option `--profile no-profile`.
 
-Any parameter used by iidy can be set using IIDY_{{argname}}
+## Additional Documentation
 
-An example of this would be chaning the default environment from development to production.
-
-```
-export IIDY_ENVIRONMENT=production
-```
+- [Converting Existing CloudFormation Stacks to iidy](docs/converting-existing-cloudformation-stacks-to-iidy.md)
+- [Custom Resource Templates](docs/custom-resource-templates.md)
+- [Using Parameter Store](docs/using-parameter-store.md)
+- [Working with Non-CloudFormation YAML](docs/working-with-non-cloudformation-yaml.md)
+- [Working with StackSets](docs/working-with-stacksets.md)
+- [YAML Processing](docs/yaml-preprocessing.md)
 
 ## Examples
-See the examples/ directory.
 
+See the [`examples/`](examples/) directory.
 
 ## Development
 
-iidy is coded in Typescript and compiles to es2015 Javascript using
-commonjs modules (what Node uses). See the `Makefile` and the script
-commands in `package.json` for details about the build process.
+iidy is coded in Typescript and compiles to ES2015 JavaScript using commonjs
+modules (what Node uses). See the `Makefile` and the script commands in
+`package.json` for details about the build process.
 
-Please format all files with `tsfmt` and remove extra whitespace
-before submitting a PR.
+Please format all files with `tsfmt` and remove extra whitespace before
+submitting a PR.
 
 ## License
 
 MIT.
 
-## Preparing a Release
+## Releasing
```

Comment:
This is only relevant for Unbounce developers - should we demote it to the wiki?

Reply:
I think it is ok to leave here – I think it is pretty standard for release notes in a open source project to only be relevant for the maintainers of the project. It doesn't involve any Unbounce-private repositories or other resources.